settings.gradle

```diff
@@ -20,15 +20,15 @@ enableFeaturePreview('GRADLE_METADATA')
 include ':common'
 include ':jvm'
 include ':js'
-include ':native'
+//include ':native'
 
 include ':configparser'
 include ':benchmark'
 
 project(':common').projectDir = file('./runtime/common')
 project(':jvm').projectDir = file('./runtime/jvm')
 project(':js').projectDir = file('./runtime/js')
-project(':native').projectDir = file('./runtime/native')
+//project(':native').projectDir = file('./runtime/native')
```

Comment:
Accidental?

Reply:
No, mentioned bootstrap build of kotlin compiler does not compatible with K/N. When corresponding plugin changes for this PR would be merged into the release branch, I'll wait for K/N to build from that branch and rollback changes with proper versions. Native was tested on my machine with a local build of compiler.